app/assets/stylesheets/application.scss

```diff
@@ -1,4 +1,5 @@
  @import 'govuk-elements';
+ @import 'prism';
 
 .container {
   @extend %site-width-container;
```

Comment:
I think we accept this throughout of codebases i.e. https://github.com/alphagov/product-page-example/blob/master/source/stylesheets/_core.scss#L56